apps/dashboard-e2e/src/describer/fairness/singleModelView/describeSingleModelViewWithError.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getSpan } from "../../../util/getSpan";
+import { ScatterChart } from "../../../util/ScatterChart";
+import { IFairnessMetadata } from "../IFairnessMetadata";
+import { getToModelComparisonPageWithDefaults } from "../utils";
+
+export function describeSingleModelViewWithError(
+  data: IFairnessMetadata
+): void {
+  describe("single model view", () => {
+    const props = {
+      chart: undefined as unknown as ScatterChart
+    };
+    beforeEach(() => {
+      // move to the single model view
+      getToModelComparisonPageWithDefaults()
+        .get("#FairnessPerformanceTradeoffChart .trace.scatter:eq(0) .points")
+        .then(() => {
+          // click on model point in model comparison view
+          props.chart = new ScatterChart("#FairnessPerformanceTradeoffChart");
+          props.chart.clickNthPoint(0);
+        });
+    });
+    it("should find single model view", () => {
+      getSpan("Assessment results").should("exist");
+
+      cy.get("#sensitiveFeatureDropdown").should(
```

Comment:
all of these are common with the "normal" check (without "WithError" in the file name). Can't we just add
```
if (checkErrorBars) {
    <check>
}
```
and keep it in a single test?

Reply:
same applies in the model comparison file